pkg/docker/util.go

```diff
@@ -16,8 +16,8 @@ import (
 	"github.com/openshift/source-to-image/pkg/util/user"
 )
 
-// DockerImageReference points to a Docker image.
-type DockerImageReference struct {
+// ImageReference points to a Docker image.
+type ImageReference struct {
```

Comment:
Stuttering:

```console
$ cat stutter.go 
package stutter

type StutterMasterExample struct {}
$ golint stutter.go 
stutter.go:3:6: exported type StutterMasterExample should have comment or be unexported
stutter.go:3:6: type name will be used as stutter.StutterMasterExample by other packages, and that stutters; consider calling this MasterExample
```

I'm also reluctant to renames, but since this is really just internal functionality I judge it was a fine cleanup (but still a long way to go to get rid of all reasonable/actionable lint suggestions).

What we could and I don't see us using is `internal/` directories to hold internal-only packages, would maybe make our life more in peace here when refactoring internal stuff that needs to be exported for use in other packages, while not meant to be part of a public API.